CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ### Bug Fixes
 
-* [ObjectServer] `metatable` might not be created for sync 2.0.0 (https://github.com/realm/realm-object-store/issues/502).
+* [ObjectServer] Realm no longer throws a native “unsupported instruction” exception in some cases when opening a synced Realm asynchronously.
```

Comment:
How about adding a link to original issue like: (realm/realm-object-store#502) ?